### ISSUE TITLE
[Docs] Introduce the `test --reporter=html` command prior to `show-report` command

### ISSUE THREAD

**Issue by cnishina** [microsoft/playwright#20152]
### Issue I ran into

While going through the running-tests-js.md file, I tried to run the `show-report` command. I got an error:

```
npx playwright show-report
No report found at "/src/experiment-playwright-trace-viewer/playwright-report"
```

The instructions did not introduce the `--reporter=html` option  prior to running `show-report` command.

### What should happen

Show the `npx playwright test --reporter=html` command prior to the `npx playwright show-report` command.

**Comment by debs-obrien** [comment]
Hi @cnishina 

For projects created with `create-playwright` the HTML report is enabled by default. If you have a config which does not use the HTML report or running the show-report command yields to no reports, you can run it with `--reporter=HTML`.

I have added this comment to the docs at the very end as this is a getting started guide the majority of people should be following this guide from the start and will therefore have Playwright installed with create-playwright and therefore won't run into this issue. They can just use `npx playwright show-report` with no flag needed. I prefer not to have everyone use this flag as default when it is not necessary.

For those that are more advanced and have played with the config or have an already installed project that they are working on then they may come across this and therefore have added the comment above. 

You can see pr here which also has some other minor improvements: https://github.com/microsoft/playwright/pull/20148/files

**Comment by debs-obrien** [comment]
closed via #20148
